package.json

```diff
@@ -33,7 +31,10 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.23.10",
     "@babel/plugin-transform-class-static-block": "^7.26.0",
-    "@nrwl/nx-cloud": "^15.3.5",
+    "@nx/js": "^20.7.1",
+    "@swc-node/register": "^1.9.1",
+    "@swc/core": "^1.5.7",
+    "@swc/helpers": "^0.5.11",
```

Comment:
@nx/js (and these `swc` packages) are required to use `nx release version`. These were auto-installed when I ran `nx add @nx/js`